packages/kununu-utils/kununu-logger/index.js

```diff
@@ -32,16 +41,26 @@ export const formatNodeRequest = ({req, res, label, timeTakenMicros}) => JSON.st
  * on the custom param.
  */
 export const customFormat = printf((info) => {
-  const colorizedMessage = getColorizedMessage(`[${info.label}][${info.timestamp}][${info.level}]`);
-  const loggerType = info.custom ? '– custom logger -' : ' – middleware logger - ';
-  const kibanaFormatting = process.env.NODE_ENV === 'production';
+  const loggerType = info.custom ? 'custom_logger' : 'middleware_logger';
+  const colorizedMessage = getColorizedMessage(`[${info.label}][${info.timestamp}][${info.level}][${loggerType}]`);
 
-  // Kibana only supports JSON and not text so the prefix only added
-  // on non kibana formatting
-  const prefix = kibanaFormatting ? '' : `${colorizedMessage}${loggerType}`;
+  // Logs in production should be outputted in JSON and not text
+  const prefix = (process.env.NODE_ENV === 'production') ? '' : `${colorizedMessage}`;
 
   if (info.custom) {
-    return `${prefix}${JSON.stringify({...info, logType: 'custom_logger', build: process.env.BUILD_NAME || '-'})}`;
+    return `${prefix}${JSON.stringify(
+      {
+        message: info.message,
```

Comment:
This is partially the same as before, can this be abstracted? So that we don't need to change in both places if the time format changes, for example...